### PR TITLE
Add filtering of log messages by group - part 2

### PR DESCRIPTION
**Pull request by daniel-thom**
I found a few missing items when implementing log filtering support in PSY.

### CODE REVIEW

File: src/utils/logging_config.toml [hunk 1]
```diff
@@ -8,10 +8,10 @@ file_mode = "w+"
 set_global = true
 
 ["group_levels"]
-# Set these to Info to suppress the group when debug logging is enabled.
+# Uncomment these to suppress the group when debug logging is enabled.
```
Comment (jd-lara) [inline src/utils/logging_config.toml:11]: Can you entries to the documentation about this in the PR you have some time? I think this is useful but we need to have a good explanation of this for the users (including us) to actually exploit it.

Comment (daniel-thom) [reply src/utils/logging_config.toml:11]: Did you see these updates in the last PR?
- https://nrel-siip.github.io/InfrastructureSystems.jl/stable/dev_guide/logging/#Use-Cases
- https://nrel-siip.github.io/InfrastructureSystems.jl/stable/dev_guide/tests/

Comment (jd-lara) [reply src/utils/logging_config.toml:11]: Yes, I meant on how to modify the TOML file to achieve certain behaviors

Comment (daniel-thom) [reply src/utils/logging_config.toml:11]: I added more documentation about what we are doing with these groups but am not sure I addressed your concerns. Aren't the comments in this file self-explanatory?

**Review by jd-lara — APPROVED** [review]

**Pull request merged**